core/node/libp2p/rcmgr_defaults_test.go

```diff
@@ -0,0 +1,131 @@
+package libp2p
+
+import (
+	"testing"
+
+	rcmgr "github.com/libp2p/go-libp2p-resource-manager"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestResourceManagerDefaults(t *testing.T) {
+	require := require.New(t)
+	require.Equal(expectedDefaultLimits, rcmgr.DefaultLimits, "===> OOF! go-libp2p-resource-manager changed DefaultLimits")
+}
+
+// https://github.com/libp2p/go-libp2p-resource-manager/blob/v0.1.5/limit_defaults.go#L49
```

Comment:
Is there a clean way (maybe not) in Go to also import this version instead of duplicating its constants?

Reply:
you can scale it with `WithSystemMemory` or something like that.
Badically it gives you default limits, memory adjusted and you can modify ehat you want.